test/consumeFlow.test.ts

```diff
@@ -49,6 +49,7 @@ describe("Ocean CLI Publishing", function() {
         }
 
         process.env.PRIVATE_KEY = "0x1d751ded5a32226054cd2e71261039b65afb9ee1c746d055dd699b1150a5befc";
+        // 0x529043886F21D9bc1AE0feDb751e34265a246e47
```

Comment:
What's the purpose of this comment without any explanation?

Reply:
for the .env vars like `AUTHORIZED_PUBLISHERS` (and all the other similar vars for the roles) 
we need to deal with account addresses..
how do we know the address that corresponds to this PRIVATE KEY 
0x1d751ded5a32226054cd2e71261039b65afb9ee1c746d055dd699b1150a5befc 
?

Reply:
Ok, so perhaps a minor change can help. No harm in being extra clear.

```suggestion
        // Using this account: 0x529043886F21D9bc1AE0feDb751e34265a246e47
```